collect_app/src/androidTest/java/org/odk/collect/android/support/pages/EditSavedFormPage.java

```diff
@@ -59,7 +59,7 @@ private void closeDraftsPillsEducationDialog() {
     }
 
     public EditSavedFormPage checkInstanceState(String instanceName, String desiredStatus) {
-        int desiredImageId = InstanceListCursorAdapter.getFormStateImageResourceIdForStatus(desiredStatus);
+        int desiredImageId = InstanceListItemView.INSTANCE.getFormStateImageResourceIdForStatus(desiredStatus);
```

Comment:
They is a nice change, but I was avoiding things like this to minimise merge conflicts. I think we should look for a remove duplication as part of #5781.

Reply:
Ok should I remove that commit for now?

Reply:
Done.